Packs/Binalyze/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Binalyze AIR
+- All acquisition profiles support added.
```

Comment:
Should the code be updated with this change?

Reply:
Nice catch! I added the code change. Thank you!!